### DISCUSSION TITLE
Breakout Room 3 - Series 3

### DISCUSSION THREAD

**Discussion by Rory646** [IBAHCM/RPiR#219] [Archive]
Hello, 

The people in our group are:

Annabel Locke  - @AnnabelLocke

Samantha Keyse - @skeyse1 

Ben Chua - @BenChua1999

Rory Thomson - @Rory646

**Comment by Rory646** [comment]
Hi,

Here are Annabel and I's repos

Annabel: https://github.com/IBAHCM/annabellockeseries03

Rory: https://github.com/IBAHCM/Rory646Series03

**Reply by BenChua1999** [reply]
Thanks Rory, and this is mine, hope it works:https://github.com/IBAHCM/BenChua_Series03
